packages/backend/src/payment-method/ilp/service.ts

```diff
@@ -43,6 +43,13 @@ async function getQuote(
   deps: ServiceDependencies,
   options: StartQuoteOptions
 ): Promise<PaymentQuote> {
+  const stopTimerRates = deps.telemetry.startTimer(
+    'ilp_get_quote_rate_time_ms',
+    {
+      callName: 'rates',
+      description: 'Time to get rates'
+    }
+  )
   const rates = await deps.ratesService
     .rates(options.walletAddress.asset.code)
```

Comment:
maybe best to just convert to a normal try-catch call?

Reply:
converted from chaining to try/catch/finally